### DISCUSSION TITLE
Conda Information Not Displayed

### DISCUSSION THREAD

**Discussion by meetthehorizon** [starship/starship#6409] [Q&A] [answered]
I want to display information about conda but nothing pops up here
![image](https://github.com/user-attachments/assets/448401a3-9f74-45ef-85a1-043966b18b24)

- `echo $CONDA_DEFAULT_ENV`: `base`
- `conda config --show changeps1`: `changeps1: False`
- Config File: [starship.copy.1.txt](https://github.com/user-attachments/files/17869805/starship.copy.1.txt)

Any help is appreciated!

**Comment by davidkna** [comment, marked as answer]
Please try setting the `conda.ignore_base` starship option to `false`.
